rust/src/api/event_type.rs

```diff
@@ -101,28 +126,12 @@ impl<'a> EventType<'a> {
         .await
     }
 
-    /// Partially update an event type.
-    pub async fn patch(
-        &self,
-        event_type_name: String,
-        event_type_patch: EventTypePatch,
-    ) -> Result<EventTypeOut> {
-        event_type_api::v1_period_event_type_period_patch(
-            self.cfg,
-            event_type_api::V1PeriodEventTypePeriodPatchParams {
-                event_type_name,
-                event_type_patch,
-            },
-        )
-        .await
-    }
-
     /// Archive an event type.
     ///
     /// Endpoints already configured to filter on an event type will continue to
     /// do so after archival. However, new messages can not be sent with it
     /// and endpoints can not filter on it. An event type can be unarchived
-    /// with the create operation.
+    /// with the [create operation][Self::create].
```

Comment:
This one is a problem for the CLI. Need to surface the params so we can use `--expunge`. This is supported in the Go CLI currently.